### PR TITLE
Include Map API

### PR DESCRIPTION
**Pull request by gferioli0418**
-Include Map API in the Bottom of home page.
-Put something different for API Key
-Add inline Styling for map

### CODE REVIEW

**Comment by josueetcom** [comment]
@NzingaHolloman, please review this pull request

File: portfolio/pom.xml [hunk 1]
```diff
@@ -16,13 +16,26 @@
...
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+</dependency>
```
Comment (josueetcom) [inline portfolio/pom.xml]: Weird, I thought this was included already last week

Comment (josueetcom) [inline portfolio/pom.xml]: Please fix the indentation:
```suggestion
    <dependency>
      <groupId>com.google.code.gson</groupId>
      <artifactId>gson</artifactId>
      <version>2.8.6</version>
    </dependency>
```

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -4,6 +4,27 @@
...
+        height: 100%;
+      }
+      /* Optional: Makes the sample page fill the window. */
+      html,
+      body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+    </style>
```
Comment (josueetcom) [inline portfolio/src/main/webapp/index.html]: [Style](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns): Separate structure from presentation from behavior.

Please move the styling to your CSS file

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -48,3 +48,12 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+
+function createMap() {
+  const map = new google.maps.Map(document.getElementById("map"), {
+    center: { lat: 29.646445, lng: -82.347676 },
+    zoom: 16
```
Comment (josueetcom) [inline portfolio/src/main/webapp/script.js]: [Style](https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma): Include a trailing comma whenever there is a line break between the final property and the closing brace.

```suggestion
    zoom: 16,
```

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (josueetcom) [reply portfolio/src/main/webapp/index.html]: You may want to alphabetize CSS properties and use shorthand ones (comments from PR #8) while you're at it

**Comment by gferioli0418** [comment]
I had a few issues working with git and that's why I accidentally pushed three commits with the same names. In the last commit, I changed some styling issues.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -48,8 +33,7 @@
     ></script>
     <script src="script.js"></script>
   </head>
-
-  <body>
+  <body class = "bd" onload="createMap();">
```
Comment (josueetcom) [inline portfolio/src/main/webapp/index.html]: [Style](https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity): Use valid HTML where possible. 

Try copying/pasting this file on https://validator.w3.org/nu/#textarea

Comment (josueetcom) [reply portfolio/src/main/webapp/index.html]: Running `prettier` on the file will also show the same errors

**Comment by gferioli0418** [comment]
Fixed the missing tags issue. Thanks for the link Josue, I found it to be very useful

**Pull request merged**